packages/workflow/src/records/handler/validate.ts

```diff
@@ -34,11 +35,12 @@ export const validateRoute = createRoute({
       request.payload
     )
 
-    const validatedRecord = await toValidated(
-      record,
-      token,
-      payload.comments,
-      payload.timeLoggedMS
+    await toValidated(record, token, payload.comments, payload.timeLoggedMS)
```

Comment:
Is there an issue with `toValidated` if it doesn't return the same result an additional `getRecordById` would?

Reply:
Basically it's replacing the current task with the new one. But what we need is to transform the previous one into `TaskHistory` _and_ add the new one. I didn't go with that route as it would require some broader changes which would affect the other "flows". While with this change we only need to concern ourselves with the `Approval` flow